nengo_bones/config.py

```diff
@@ -147,7 +147,7 @@ def fill_defaults(config):  # noqa: C901
 
     if "contributors_rst" in config:
         cfg = config["contributors_rst"]
-        cfg.setdefault("nengo_list", license_type == "nengo")
+        cfg.setdefault("nengo_list", config["license"].startswith("abr"))
```

Comment:
I don't think we need this field any more right? We're just directly checking the license field in the contributors template.

Reply:
Ah yes, you're right! If you're fine with that change then yes let's remove this option.